frontend/src/pages/groups/GroupForm.tsx

```diff
@@ -30,40 +32,53 @@ interface Props {
  * This component encapsulates a form for creating and editing groups.
  */
 const GroupForm: FunctionComponent<Props> = (props) => {
+  const { profile } = useContext(UserProfileContext)
+
   const {
     register,
     handleSubmit,
     reset,
     formState: { errors },
-  } = useForm()
+  } = useForm<GroupCreateInput>()
 
   useEffect(
     function resetFormValues() {
       if (props.defaultValues) {
         // Update the values of the fields
-        reset(props.defaultValues.group)
+        reset(_omit(props.defaultValues.group, ['id', '__typename']))
```

Comment:
Suggestion (non-blocking): could extract this to a shared function called `omitGraphqlTypeName` or something

Reply:
This is the only place where we're doing that so I'll leave it as is, but I'll create a function if we end up repeating that same code in the future 👍